concent_api/core/tests/test_unit_model.py

```diff
@@ -0,0 +1,164 @@
+from django.core.exceptions import ValidationError
+import pytest
+
+from golem_messages import factories
+from golem_messages import message
+
+from core.message_handlers import store_subtask
+from core.models import Subtask
+from core.utils import hex_to_bytes_convert
+
+
+# This all data has to be prepared in separate function because pytest parametrize can't get variables from SetUp()
+# Function allows to prepare 2 packs of data: correct (default) and incorrect.
+def _get_data_list(correct=True):
```

Comment:
one more thing: remove the default value ane explicitly state whether `correct` is `True` or `False` (update the doc-string as well)

Reply:
It also done.